include/crypto/mlkem.h

```diff
@@ -160,15 +178,52 @@ extern "C" {
                             const ossl_mlkem768_private_key *private_key,
                             ossl_mlkem_ctx *mlkem_ctx);
 
-    /*
-     * ossl_mlkem768_recreate_public_key recreates a fully formed ossl_mlkem768_public_key
-     * from an input |encoded_public_key| of size ossl_mlkem768_PUBLIC_KEY_BYTES.
-     * |pub| is expected to point to an allocated memory area of
-     * sizeof(ossl_mlkem768_public_key)
-     */
-    int ossl_mlkem768_recreate_public_key(const uint8_t *encoded_public_key,
-                                          ossl_mlkem768_public_key *pub,
-                                          ossl_mlkem_ctx *mlkem_ctx);
+/* Serialisation of keys. */
```

Comment:
fwiw, this indentation caused problems with "util/check-format.pl" for me -- so I needed to indent everything by 4 to get the style check to pass...

Reply:
Ha, I've just now realized this header is indented due to the `extern "C" {}` thing. We should just remove that as there should be no `extern "C" {}` annotation in the internal headers, they should be present only in the public headers.